Core/src/main/java/com/plotsquared/core/util/ChunkManager.java

```diff
@@ -79,8 +79,8 @@ public static void setChunkInPlotArea(
         }
     }
 
-    public static boolean preProcessChunk(BlockVector2 loc, ScopedQueueCoordinator queue) {
-        final RunnableVal<ScopedQueueCoordinator> forceChunk = forceChunks.get(loc);
+    public static boolean preProcessChunk(BlockVector2 loc, ZeroedDelegateScopedQueueCoordinator queue) {
```

Comment:
Missing javadocs and since annotations

Reply:
Add javadocs if you can work out quite what Jesse was planning with these methods :))